openapi.yaml

```diff
@@ -7374,6 +7374,58 @@ paths:
       - lang: CLI
         source: >
           linode-cli managed service-enable 9994
+  /managed/stats:
+    description: |
+      A collection of managed stats.
+    x-linode-cli-command: managed
+    get:
+      x-linode-grant: unrestricted only
+      tags:
+      - Managed
+      summary: List Managed Stats
+      description: |
+        Returns a list of Managed Stats on your Account in the form of y and x graph data points.
```

Comment:
probably best to either explain what the x and y points are (and put them in x, y order instead of y, x cause it's confusing that way), or just say something like:

Returns a list of Managed Stats on your Account. These stats reflect the last 24...

Either way works for me.